Framework/Geometry/src/Instrument/Goniometer.cpp

```diff
@@ -17,6 +17,8 @@ using Kernel::DblMatrix;
 using Kernel::V3D;
 using Kernel::Quat;
 
+Mantid::Kernel::Logger g_log("Goniometer");
```

Comment:
To get it to compile on Linux, you need to add `#include "MantidKernel/Logger.h"` at the start of the file.